### PR TITLE
helper-cli: Fix a NPE in writeAsYaml() 

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

**Comment by sschuberth** [comment]
> but also the conversion of targetFile to an absolute path

So, if the real goal is to get an absolute path, we should simply call `absolutePath` IMO.

**Comment by fviernau** [comment]
> So, if the real goal is to get an absolute path, we should simply call `absoluteFile` IMO.

Done.

File: helper-cli/src/main/kotlin/common/Utils.kt [hunk 1]
```diff
@@ -55,6 +55,7 @@ import org.ossreviewtoolkit.model.yamlMapper
 import org.ossreviewtoolkit.utils.CopyrightStatementsProcessor
 import org.ossreviewtoolkit.utils.ORT_NAME
 import org.ossreviewtoolkit.utils.OkHttpClientHelper
+import org.ossreviewtoolkit.utils.expandTilde
```
Comment (sschuberth) [inline helper-cli/src/main/kotlin/common/Utils.kt:58]: This needs to go now.

File: helper-cli/src/main/kotlin/common/Utils.kt [hunk 2]
```diff
@@ -522,7 +523,8 @@ internal fun RepositoryConfiguration.sortScopeExcludes(): RepositoryConfiguratio
  * Serialize a [RepositoryConfiguration] as YAML to the given target [File].
  */
 internal fun RepositoryConfiguration.writeAsYaml(targetFile: File) {
-    targetFile.parentFile.safeMkdirs()
+    targetFile.absoluteFile.parentFile.safeMkdirs()
+
```
Comment (sschuberth) [inline helper-cli/src/main/kotlin/common/Utils.kt:527]: Consider not adding a blank line here.

Comment (fviernau) [reply helper-cli/src/main/kotlin/common/Utils.kt:527]: Done.

File: helper-cli/src/main/kotlin/common/Utils.kt (hunk 1 above)
Comment (fviernau) [reply helper-cli/src/main/kotlin/common/Utils.kt:58]: Done.

**Comment by fviernau** [comment]
> The commit message still contains "revert" at the bottom, and lacks a sign-off.

Sorry @sschuberth , fixed it.

**Pull request merged**